README.md

```diff
@@ -157,6 +158,9 @@ const Readme: Component = () => (
     <LineBreak />
     {/* Create a contributing section linking to the contributing guidelines file */}
     <ContributingSection />
+    <LineBreak />
```

Comment:
You'll need to remove this now since we made the change to include the linebreak in the sections.

Reply:
Noted.